### PR TITLE
Redesign Patient Info Card

### LINKED ISSUE #6627: Design Enhancement Scope in Patient Consultation Page ( ICD-11 Diagnosis )
**Describe the bug**

- [ ] multiple diagnosis in the consultation page is stretching the entire page, it doesn't give a good User Experience

**To Reproduce**

1. Go to Patient consultation page
2. See error

**Expected behavior**

- [ ] make a new card between the header card and the patient info card

Note: Whoever wishes to work on this issue, get the mock-up design approved in the EOD call, then only, the issue will be assigned to the respective author.

**Screenshots**

image

### LINKED ISSUE #6628: Design Enhancement Scope in Patient Consultation Page ( Patient Details )
**Describe the bug**

there is lots of white space wasted in the top side of the patient consultation page which carry patient details and related badges

**To Reproduce**

1. Go to patient consultation page
2. See error

**Expected behavior**

An effective way to utilize the spacing and a new design

Note: Whoever wishes to work on this issue, get the mock-up design approved in the EOD call, then only, the issue will be assigned to the respective author.

**Screenshots**

image

### PR DESCRIPTION
**Pull request by Ashesh3**
Fixes #6627
Fixes #6628

This pull request includes changes to the Patient Info Card component, improving its design and layout.

image
image

### CODE REVIEW

**Comment by nihal467** [comment]
@Ashesh3 
image

the log update button seems to have a huge width, can you check reducing it and whether it will look good or not ?

File: src/Components/Diagnosis/DiagnosesListAccordion.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      </div>
+      <div
+        className={classNames("transition-all duration-500 ease-in-out")}
+        style={
+          isVisible
+            ? {
+                overflow: "visible",
+              }
+            : { height: "0px", overflow: "hidden" }
+        }
```
Comment (rithviknishad) [inline src/Components/Diagnosis/DiagnosesListAccordion.tsx]: Was this not possible with tailwind?

**Review by rithviknishad — APPROVED** [review]

**Comment by rithviknishad** [comment]
Should we make the "Hide diagnoses" button less intense?

The diagnoses itself seems to be of `font-normal`. But hide diagnoses is gaining more attention with it being bold than the diagnoses itself.

File: src/Components/Diagnosis/DiagnosesListAccordion.tsx (hunk 1 above)
Comment (Ashesh3) [reply src/Components/Diagnosis/DiagnosesListAccordion.tsx]: Definitely possible 😅. I Initially reused this code from the `AccordionV2` component. I've made it work with the tailwind classes now, along with the same change for the `AccordionV2` component too.

**Comment by Ashesh3** [comment]
> Should we make the "Hide diagnoses" button less intense?
> 
> The diagnoses itself seems to be of `font-normal`. But hide diagnoses is gaining more attention with it being bold than the diagnoses itself.

I agree, it took away the initial focus of the user from the diagnoses list by standing out too much. I've softened it out a bit. Although no changes are to be made for "Show diagnoses", since it's supposed to stand out and be obvious. Let me know how it looks now.
```
- Removed semibold
+ text-gray-600 hover:bg-gray-200
```
image
image

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@Ashesh3 modify the failing test

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**